docs/Template10Example.md

```diff
@@ -0,0 +1,5 @@
+An simple example application using the Template10 package initially built and maintained by Jerry Nixon of Microsoft but with support and work across many people and departments in Microsoft. This is available as a NuGet package via tha Extensions Manager in Visual Studio with support and documentation here: http://aka.ms/Template10.
```

Comment:
Ensure that the example source folder itself has a copy of the licence for the Template10 package.